tests/components/mqtt/test_light_json.py

```diff
@@ -920,7 +459,7 @@ async def test_controlling_state_color_temp_kelvin(
         )
     ],
 )
-async def test_controlling_state_via_topic2(
+async def test_controlling_state_via_topic(
```

Comment:
Renamed because the old `test_controlling_state_via_topic` test used the removed legacy `color_mode` support.